### PR TITLE
first microverse project hello microverse

### PR DESCRIPTION
**Pull request by jossi30**
a simple project to demonstrate the use of linters and github workflow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse</h1>
+</body>
```
Comment (Qoosim) [inline index.html:12]: - [ ] It is required that a class name should be given to the `h1` and use that class name in your css file to style the content of the `h1` 

class-name

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: navy;
+}
```
Comment (Qoosim) [inline style.css:3]: - [ ] It is best practice to use class name instead element selector. Kindly give a class name to your `h1` tag and use that class name here to style the content of the `h1`.

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,5 @@
+test.md
```
Comment (Qoosim) [inline .gitignore:1]: - [ ] I can see that you've added this file to the `.gitignore` but it still reflects on your repository. Follow any of this files to rectify the issue. [This file](https://www.atlassian.com/git/tutorials/saving-changes/gitignore)  or [this file](https://www.pluralsight.com/guides/how-to-use-gitignore-file)

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @jossi30, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 -   No linter errors. :heavy_check_mark: 
 -   Good README file. :heavy_check_mark: 
 -   Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jossi30** [comment]
added the requirements asked by the reviewer

**Review by Olamarx — APPROVED** [review]
Hi @jossi30, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All linters passed ✅ 
- Git ignores test.md ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**